src/EnergyPlus/GroundTemperatureModeling/SiteDeepGroundTemperatures.cc

```diff
@@ -69,20 +69,11 @@ std::shared_ptr<SiteDeepGroundTemps> SiteDeepGroundTemps::DeepGTMFactory(EnergyP
     // SUBROUTINE INFORMATION:
     //       AUTHOR         Matt Mitchell
     //       DATE WRITTEN   Summer 2015
-    //       MODIFIED       na
-    //       RE-ENGINEERED  na
 
     // PURPOSE OF THIS SUBROUTINE:
     // Reads input and creates instance of Site:GroundTemperature:Deep object
 
-    // USE STATEMENTS:
-    using namespace GroundTemperatureManager;
-
-    // Locals
     // SUBROUTINE LOCAL VARIABLE DECLARATIONS:
-    int NumNums;
-    int NumAlphas;
-    int IOStat;
```

Comment:
[src/EnergyPlus/GroundTemperatureModeling/SiteDeepGroundTemperatures.cc:83]:(style),[variableScope],The scope of the variable 'NumNums' can be reduced.
[src/EnergyPlus/GroundTemperatureModeling/SiteDeepGroundTemperatures.cc:84]:(style),[variableScope],The scope of the variable 'NumAlphas' can be reduced.
[src/EnergyPlus/GroundTemperatureModeling/SiteDeepGroundTemperatures.cc:85]:(style),[variableScope],The scope of the variable 'IOStat' can be reduced.